aries-site/src/pages/components/table.mdx

```diff
@@ -14,9 +14,11 @@ import {
 } from '../../examples';
 
 <Example
+  componentName="Table"
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/table/TableExample.js"
   docs="https://v2.grommet.io/datatable?theme=hpe#props"
   figma="https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table-Component?node-id=195%3A0"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Table/Table.js"
```

Comment:
What we call "Table" on the Design System site is really a DataTable. I think we should link to the DataTable component here.

Also, I think we should probably relabel this page as DataTable for parity or at least have a discussion about it. @jcfilben don't let this hold up this PR, will you please add an issue in the backlog to track?

Reply:
Tangentially, we have a "Search" component on the Design System, but it is really a TextInput with Search behavior. I believe we did this so that it provided dedicated focus to a search execution. This still feels right to me, but could potentially move to Templates/Patterns instead of Components to keep parity, as it is a specialized pattern...  Thoughts? If not, should we leave a "view Search source" off of the Search page or link to the TextInput?

Reply:
I changed the link in Table to go to DataTable. I will open an issue about relabeling the page as DataTable.

I added a link to the TextInput source on the Search page. I think since we have a link to the TextInput docs already on the Search example it makes sense to also link to the source code. I'm fine with leaving Search in the components section